example.ini

```diff
@@ -43,6 +43,11 @@ base-url =
 ; secret token for communication with harold
 secret =
 
+[elasticsearch]
```

Comment:
can you add some comments here explaining what elasticsearch is used for and what the specific configurables are?